examples/chart_combined.pl

```diff
@@ -67,11 +67,12 @@
 
 #
 # In the second example we will create a similar combined column and line
-# chart except that the secondary chart will have a secondary Y axis.
+# chart except that the secondary chart will have a secondary Y axis,
```

Comment:
Omit this change to the example program. Worksheet charts are rarely used by end users of the module.